coderd/parameter/renderer_test.go

```diff
@@ -47,3 +47,40 @@ __This is bold text.__
 		require.Equal(t, nothingChanges, stripped)
 	})
 }
+
+func TestHTML(t *testing.T) {
```

Comment:
Can you add a test for script injection?
e.g. `<p onclick="alert(\"omghax\")">Click here to get access!</p>`?